tests/functional/preview_and_dev/test_cbc_integration.py

```diff
@@ -198,6 +201,7 @@ def test_broadcast_with_az1_failure_tries_az2(driver, api_client):
     )
 
     responses = db_response["Items"]
+    print(responses)
```

Comment:
There are a few prints left in here which will always print, should these only be output on test failure?

Reply:
Removed